packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -16,8 +16,8 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: foreman >= 1.17.0
 
 Requires: %{?scl_prefix}rubygem(foreman-tasks-core)
-Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.29
-Requires: %{?scl_prefix}rubygem(dynflow) < 0.9.0
+Requires: %{?scl_prefix}rubygem(dynflow) >= 1.0.0
```

Comment:
The `BuildRequires` is still pointing to `>= 0.8.29`, please update that one too.
 Also I believe the line below is incorrect, although desirable (in my opinion), it's not in the gemspec.

Reply:
If you mean the `< 2.0.0` then that's correct. `~> 1.0` implies `< 2.0`.